### PR TITLE
Removed grpcio direct requirement

### PR DESCRIPTION
**Pull request by MuratUrsavas**
As this package creates a lot of long time build issues I've removed direct dependency from here to let parent app decide which grpcio version to use.

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

**Review by kashifpk — APPROVED** [review]

File: test-requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ pytest==7.2.0
 pytest-cov==4.0.0
 flake8==6.0.0
 responses==0.22.0
+grpcio==1.53.0
```
Comment (pritamghanghas) [inline test-requirements.txt:6]: why not fix it in requirements itself. Kind of missing dependency.

**Comment by shawaj** [comment]
Wouldn't it be better to just use a >= here instead of removing it altogether? (Or just no specific version and just using grpcio)

Because it is a dependency and we surely need at least some version of it?

**Comment by shawaj** [comment]
@MuratUrsavas it seems minimum we are using is 1.50.1 so can't we just say grpcio>=1.50.1 ?

File: test-requirements.txt (hunk 1 above)
Comment (MuratUrsavas) [reply test-requirements.txt:6]: @pritamghanghas That's on purpose. I've removed this dependency so I can use the most convenient one in the target container. For hm-gatewayrs, it's 1.50.1, hm-diag, 1.53.0 so on so forth. This way we can avoid unnecessary grpcio builds.

**Pull request merged**